spec/features/book_index_spec.rb

```diff
@@ -50,12 +53,12 @@
   end
 
   it "user_can_see_all_authors" do
-    author_1 = @book_1.reviews.create(title: "Good book", score: 4, review_text: "text body")
-
+    book_4 = @author_1.books.create(title: "Hamlet", pages: 342, year_pub: 1992, image: "https://images-na.ssl-images-amazon.com/images/I/41IETeONh-L._SX331_BO1,204,203,200_.jpg", publisher: "Simon & Schuster")
+    # binding.pry
     visit "/books"
 
-    expect(page).to have_content(@book_1.author)
-    expect(page).to have_content(@book_2.author)
-    expect(page).to have_content(@book_3.author)
+    expect(page).to have_content(@book_4.name)
+    # expect(page).to have_content(@book_2.author)
```

Comment:
We can do (I'm pretty sure) a @books.authors.each within the tests to test that we expect each/all authors from each book